src/services/pubnub/pubnub_service.js

```diff
@@ -0,0 +1,31 @@
+import { serviceResponse } from '../../util';
```

Comment:
we got rid of pubnub - this file should be gone now

Reply:
ugh, i definitely messed up my rebase somewhere along the lines.  same thing happened with sqs, and i just ended up deleting manually.

the following will be removed:
* sqs
* pubnub
* slack